fedimint-client/src/module/mod.rs

```diff
@@ -287,6 +288,12 @@ pub trait PrimaryClientModule: ClientModule {
         operation_id: OperationId,
         amount: Amount,
     ) -> ClientOutput<<Self::Common as ModuleCommon>::Output, Self::States>;
+
+    async fn await_primary_module_output_finalized(
+        &self,
+        operation_id: OperationId,
+        out_point: OutPoint,
+    ) -> anyhow::Result<()>;
```

Comment:
I wonder if this should return the amount received? Is that useful or a foot-gun?

Reply:
I think that would be fine